book/content/very_large_datasets/very_large_datasets.md

```diff
@@ -0,0 +1,56 @@
+
+As time passes and technologies advance and change, it will become important that these data-sharing infrastructures are expandable to include new formats, workflows, and genres of data. Given the current trends in dataset size, it will also be important for data-sharing infrastructures to be able to accommodate larger and larger datasets. 
+
+Ideally, data-sharing infrastructures can help make open research practices more accessible by hosting a list of Frequently Asked Questions, or by providing teaching tools to help new contributors and new users figure out how to use the infrastructure. The FAQ and teaching tools can also bridge gaps between language and ways of thinking that arise between areas of expertise. Discussion forums can provide opportunities for user interactions and collaborations, nurturing an open research community. 
+
+Lastly, a data-sharing infrastructure should have measures in place to prevent unfair or bad faith use of the data. 
+
+ Refs for this section: 
+ - https://gking.harvard.edu/files/gking/files/dvn.pdf
+ - https://crcns.org/files/news/crcns-neuroinformatics-article.pdf
+
+## How should researchers prepare their Very Large Datasets for sharing?
+
+The following is a suggested check-list for meta-data to include when researchers are preparing to share a Very Large Dataset:
+ - Description of the theoretical context and experimental paradigm
+ - Description of experimental conditions, ideally a fully descriptive data collection protocol
+   - date and time of data collection
+   - researchers involved in data collection
+   - species, age, sex, etc of experiment participants/subjects
+   - housing conditions / home context of experiment participants/subjects
+   - tools and techniques used to collect data (electrode type, video camera model, tissue staining technique, microphone type, etc)
+   - surgical procedures involved, if any
+   - if stimuli were used, how were the stimuli generated?
+   - how timings of stimuli, recordings, and any other events in the experiment were correlated and/or synchronized
+ - Known caveats about the use of the data (i.e. identified artifacts)
+ - links to publications
+ - tools and procedures used to save, archive, transport, process, analyse, and visualize the data (see section on Data Provenance)
+
+## Data Provenance
+
+The word "[provenance](https://en.wikipedia.org/wiki/Provenance)" refers to the chronology (or timeline) of origin, ownership, custody, or location of a historical object, originally used in relation to works of art. If someone has custody of an object, this means that that person is responsible for the keeping of object safe and in good condition. 
```

Comment:
Potential additional sources for this section: https://doi.org/10.1007/978-0-387-39940-9_1305

This page seems to have a nice section on why data provenance is important which could be an interesting sub-section: https://www.ands.org.au/working-with-data/publishing-and-reusing-data/data-provenance and then gets really technical...

Reply:
Nice! How deep would you want the data provenance section to be for Turing Way? A high-level overview, or some practical steps to implement into one's daily practice as a researcher...?